src/v2/models/AppState.js

```diff
@@ -12,6 +12,7 @@
 
 import { _, Model } from 'okta';
 import Logger from 'util/Logger';
+import RemediationEnum from '../ion/RemediationEnum';
```

Comment:
Adding onto `treeshaking` , it would be better to import only what is necessary like so -
`import { NEED_HIDE_CANCEL_FORMS } from '../ion/RemediationMap'`